lib/tictactoe_ui.ex

```diff
@@ -1,88 +1,175 @@
 defmodule TicTacToe.UI do
 
-  def start_game() do
-    TicTacToe.make_move()
-    |> decide_turn_type()
-  end
-  
-  def get_move(game_status) do
-    print_turn(:get_player_move, game_status) 
-  end
+  @clear_screen "\e[H\e[2J"
+
+  def start_game_loop, do: start_game |> game_loop
+
+  def start_game, do: TicTacToe.first_game_status |> do_next_turn
```

Comment:
I'm a little confused by this.

start_game_loop calls start_game, which not only starts the game but also 'does the next turn', which sounds suspiciously like a first round of the game_loop.

You might be able to simplify this.